src/case-swapping.js

```diff
@@ -15,7 +15,17 @@
  * @return {string}
  */
 function swapCase(phrase) {
-  // write code here
+  let res = String();
```

Comment:
make it easy
```suggestion
  let res = '';
```

what is res?
```suggestion
  let swappedPhrase = '';
```